cellpack/test-recipes/test_compartment.json

```diff
@@ -12,17 +12,17 @@
         "saveResult": true,
         "runTimeDisplay": false,
         "overwritePlaceMethod": false,
-        "innerGridMethod": "jordan3",
+        "innerGridMethod": "raytrace",
         "boundingBox": [
             [
-                -600,
-                -1100,
-                -150
+                -0.600,
```

Comment:
Is the bounding box recalculated during packing? This box looks pretty small compared to the compartments/ingredients in the recipe.

Reply:
Yes it gets expanded to include the compartment, but the compartment is really small